Source/WebInspectorUI/UserInterface/Views/InlineSwatch.js

```diff
@@ -221,7 +215,7 @@ WI.InlineSwatch = class InlineSwatch extends WI.Object
 
     _allowChangingColorFormats()
     {
-        return this._shiftClickColorEnabled && !this._readOnly && !this.value.isOutsideSRGB();
+        return !this._preventChangingColorFormats && !this._readOnly && !this.value?.isOutsideSRGB();
```

Comment:
Is it possible for there to not be a `this.value`?

Reply:
Yes, but only because I was being silly. I was calling this function before we set `this.value`.